psamm/datasource/native.py

```diff
@@ -444,8 +455,7 @@ def parse_reaction(reaction_def, context=None):
     """
 
     reaction_id = reaction_def.get('id')
-    if reaction_id is None:
-        raise ParseError('Reaction ID missing')
+    checkid(reaction_id)
```

Comment:
The `checkid` function is applied to a reaction ID but the error says "Compound ID ...".